tools/win_installer/misc/depcheck.py

```diff
@@ -12,15 +12,16 @@
 Execute with the build python, will figure out the rest.
 """
 
-import subprocess
 import os
+import subprocess
 import sys
 from multiprocessing import Process, Queue
 
 import gi
-gi.require_version("GIRepository", "2.0")
```

Comment:
This must be kept there (before from gi import GIRepository).
See http://pygobject.readthedocs.io/en/latest/guide/api/api.html#gi.require_version